Configurateur/front/lib/components/custom_header.dart

```diff
@@ -0,0 +1,260 @@
+                      child: VerticalDivider(
+                        thickness: 2,
+                        color: Color.fromARGB(255, 172, 167, 167),
+                      ),
+                    ),
+                    const SizedBox(width: 10),
+                    TextButton(
+                      onPressed: () {
+                        context.go("/company");
+                      },
+                      style: TextButton.styleFrom(
+                        foregroundColor:
+                            Provider.of<ThemeService>(context).isDark
+                                ? darkTheme.secondaryHeaderColor
+                                : lightTheme.secondaryHeaderColor,
+                        padding: EdgeInsets.zero,
+                      ),
+                      child: const Text(
+                        'Notre équipe',
+                        style: TextStyle(
+                          fontFamily: 'Inter',
+                          fontSize: 20,
+                        ),
+                      ),
+                    ),
+                    const SizedBox(width: 10),
+                    const SizedBox(
+                      height: 24,
+                      child: VerticalDivider(
+                        thickness: 2,
+                        color: Color.fromARGB(255, 172, 167, 167),
+                      ),
+                    ),
+                    const SizedBox(width: 10),
+                    TextButton(
+                      onPressed: () => goToCreation(),
+                      style: TextButton.styleFrom(
+                        foregroundColor:
+                            Provider.of<ThemeService>(context).isDark
+                                ? darkTheme.secondaryHeaderColor
+                                : lightTheme.secondaryHeaderColor,
+                        padding: EdgeInsets.zero,
+                      ),
+                      child: const Text(
+                        'Créer un conteneur',
+                        style: TextStyle(
+                          fontFamily: 'Inter',
+                          fontSize: 20,
+                        ),
+                      ),
+                    ),
+                    const SizedBox(width: 10),
+                    const SizedBox(
+                      height: 24,
+                      child: VerticalDivider(
+                        thickness: 2,
+                        color: Color.fromARGB(255, 172, 167, 167),
+                      ),
+                    ),
+                    const SizedBox(width: 10),
+                    Text("Mode sombre",
+                        style: TextStyle(
+                          fontFamily: 'Inter',
+                          fontSize: 20,
+                          color: Provider.of<ThemeService>(context).isDark
+                              ? darkTheme.secondaryHeaderColor
+                              : lightTheme.secondaryHeaderColor,
+                        )),
```

Comment:
Maybe add , after the first ) so that the file is more readable

Reply:
done